src/sensor_data_dict.py

```diff
@@ -0,0 +1,42 @@
+from sensor_temp import get_temp
+from sensor_motion import get_motion
+import alarm
+import button_home
+from time import sleep
+import datetime
+
+def create(alarm_status, button_home_status):
+    """Create data from multiple sensors
+
+    Raises:
+        e: Failed to get data from sensor
+
+    Returns:
+        dict: Dictionary of sensor data
+    """
```

Comment:
Missing `Args:`.
https://google.github.io/styleguide/pyguide.html#383-functions-and-methods

Reply:
You add this section.   Please let us know exactly how you'd like us to type it.